aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,22 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
+When using CheckBox outside of Formfield, it does not have a border or label. The hover indentifier will also be on the CheckBox itself.
+Here are some common use cases for when a CheckBox can be used outside of a FormField:
+1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.
+2. There is no additional information or help text necessary for the user to understand what is being asked of them.
```

Comment:
I am having a hard time picturing this. Can someone provide me an example for where this might occur?

Reply:
are you referring to number 2?